### ISSUE TITLE
Argument header should be allowed to be an array of integers

### ISSUE THREAD

**Issue by MaxandreJ** [JuliaData/CSV.jl#343]
Hi,

Thank you for the work you have made on this project. This is an enhancement request.

At the moment the argument `header` for the functions `CSV.read` or `CSV.File` can only be an `Int` or a `UnitRange`. I think it should also be allowed to be an array of integers (`Array{Int,1}`). This is possible in [`Pandas.read_csv`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html).

In my case, a line in the header of my `.csv` file is empty, that is interpreted as `missing`, which eventually results in the whole header being considered as `missing`. Annoying! I may have to PyCall Pandas...

Best,
Max

**Comment by MaxandreJ** [comment]
If someone wants to find a workaround for this using `Pandas.jl` I'll just leave my code here (found after a few hours):
```
import Pandas
import DataFrames
import PyCall

array_of_header_row_numbers_0_based = array_of_header_row_numbers_1_based .- 1

my_df = Pandas.read_csv(YourCSVFilePath, header = PyCall.PyVector(array_of_header_row_numbers_0_based), delimiter = "\t")

new_columns = [join(column, "_") for column in Pandas.columns(my_df)]

my_df_py = PyCall.PyObject(my_df)
my_df_py[:columns] = new_columns

my_df_jl = DataFrames.DataFrame(Pandas.DataFrame(my_df_py))
```

**Comment by MaxandreJ** [comment]
I have just updated the previous code as it had a bug.

**Comment by quinnj** [comment]
Support for this is added in #395

**Comment by MaxandreJ** [comment]
Thank you!!

**Comment by quinnj** [comment]
Fixed in #395
